k8s/upgrades/jiva-0.6.0-0.8.1/upgrade_sc.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+###############################################################################
+# STEP: Get Storage Classes                                                   #
+###############################################################################
+
+# Get the list of storageclasses, delimited by ':'
+sc_list=`kubectl get sc \
+ -o jsonpath="{range .items[*]}{@.metadata.name}:{end}"`
+rc=$?;
+if [ $rc -ne 0 ]; 
+then 
+  echo "ERROR: $rc"; 
+  echo "Please ensure `kubectl` is installed and can access your cluster."; 
+  exit; 
+fi
+
+echo "Check if openebs storage class parameters are moved to config annotation"
+for sc in `echo $sc_list | tr ":" " "`; do
+    pt="";pt=`kubectl get sc $sc -o jsonpath="{.provisioner}"`
+    if [ "openebs.io/provisioner-iscsi"  == "$pt" ];
+    then
+       uc="";uc=`kubectl get sc $sc -o jsonpath="{.metadata.labels.openebs\.io/cas-type}"`
+       if [ ! -z $uc ]; then 
+         echo "SC $sc already upgraded";
+         continue
+       fi
+
+       echo "Upgrading SC $sc";
+
+       replicas=`kubectl get sc $sc -o jsonpath="{.parameters.openebs\.io/jiva-replica-count}"`
+       pool=`kubectl get sc $sc -o jsonpath="{.parameters.openebs\.io/storage-pool}"`
+       monitoring=`kubectl get sc $sc -o jsonpath="{.parameters.openebs\.io/volume-monitor}"`
+
+       if [ -z replicas ]; then replicas="3"; fi
+       sed "s/@jiva-replica-count[^ \"]*/$replicas/g" sc.patch.tpl.yaml > sc.patch.tpl.yaml.0
+
+       if [ -z pool ]; then replicas="default"; fi
```

Comment:
this should be pool="default", and, similarly, next one monitoring="true"

Reply:
done